### PR TITLE
Fixes #3552 - disable debug console and editorial

### LINKED ISSUE #3552: Consider adding how to hide the "Debug Console" in VSCode to the ISE Compatibility doc
I just saw this tweet:
https://twitter.com/Tyriar/status/1085183089712685057?s=19

Which shows that you can hide panes at the bottom. 

If the user is only using the PowerShell extension for VSCode (or rather... Is only working on PowerShell scripts in vscode), It might be worth adding a small line or 2 to the ISE Compatibility doc under the vscode section to say:

If you only plan on using VSCode for PowerShell scripting, you can hide the "Debug Console" since it is not used by the PowerShell extension. To do so, right click on "Debug Console" and click on the check mark to hide it.

What we've found is that occasionally users get tripped up while debugging PowerShell scripts by getting confused if they should be using the Debug Console or the PowerShell Integrated Console.

Thoughts?
@rjmholt @rkeithhill @seeminglyscience @Stevel-MSFT @sydneyhsmith

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes [AB#1705773](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1705773) - Fixes #3552 - disable debug console and editorial
## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [x] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md [hunk 1]
```diff
@@ -21,161 +19,148 @@ to make the user experience a bit more familiar compared to the ISE.
...
+To do this, open the command palette (<kbd>F1</kbd> OR <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
+OR <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on macOS) and type in "ISE Mode". Select
+"PowerShell: Enable ISE Mode" from the list.
 
-This command will apply a lot of the settings found in this document automatically.
-The result looks like this:
+This command automatically applies the settings described below The result looks like this:
 
-![ISE Mode](media/How-To-Replicate-the-ISE-Experience-In-VSCode/3-ise-mode.png)
+![ISE Mode](media/How-To-Replicate-the-ISE-Experience-In-VS Code/3-ise-mode.png)
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:28]: This image is broken with the space in the URL.

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md [hunk 2]
```diff
@@ -21,161 +19,148 @@ to make the user experience a bit more familiar compared to the ISE.
...
+  below:
 
-This will hide the "Activity Bar" and the "Debug Side Bar" sections below inside of the red box:
+  ![highlighted section includes Activity Bar and Debug Side Bar](media/How-To-Replicate-the-ISE-Experience-In-VS Code/1-highlighted-sidebar.png)
 
-![highlighted section includes Activity Bar and Debug Side Bar](media/How-To-Replicate-the-ISE-Experience-In-VSCode/1-highlighted-sidebar.png)
+  The end result looks like this:
 
-The end result looks like this:
+  ![Simplified view of VS Code](media/How-To-Replicate-the-ISE-Experience-In-VS Code/2-simplified-ui.png)
```
Comment (DCtheGeek) [inline reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:63]: This image is also broken.

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:28]: fixed

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:63]: fixed

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md [hunk 3]
```diff
@@ -21,161 +19,148 @@ to make the user experience a bit more familiar compared to the ISE.
...
-> This setting is included in ["ISE Mode"](#ise-mode).
+  ```json
+  "powershell.integratedConsole.focusConsoleOnExecute": false
+  ```
 
-## No focus on console when executing
+  The default is `true` for accessibility purposes.
 
-To keep the focus in the editor when you execute with <kbd>F8</kbd>:
+  - Don't start integrated console on startup
```
Comment (TylerLeonhardt) [inline reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:83]:
```suggestion
- Don't start integrated console on startup
```

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md [hunk 4]
```diff
@@ -21,161 +19,148 @@ to make the user experience a bit more familiar compared to the ISE.
...
 
-```json
-"workbench.colorTheme": "PowerShell ISE",
-```
+- Open in the ISE
 
-> [!NOTE]
-> This setting is included in ["ISE Mode"](#ise-mode).
+  If you end up wanting to open a file in the ISE anyway,
+  you can use <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>.
```
Comment (TylerLeonhardt) [inline reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:127]: This keybinding doesn't exist anymore... but it's still in the command pallet:
image

Comment (sdwheeler) [reply reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:127]: fixed

File: reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md (hunk 3 above)
Comment (sdwheeler) [reply reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md:83]: Fixed

**Pull request merged**